lib/main.coffee

```diff
@@ -32,3 +44,7 @@ module.exports =
 
     newColumns = undefined if newColumns.length is 0
     atom.config.set('wrap-guide.columns', newColumns)
+
+  toggle: ->
+    currentState = atom.config.get('wrap-guide.enabled')
```

Comment:
Just to check, are we sure we want this to persist across sessions / windows? I'm just thinking of the behavior of Toggle Soft Wrap, which doesn't edit `config.cson`.